### DISCUSSION TITLE
499 errors: Request body not fully received

### DISCUSSION THREAD

**Discussion by dan19** [deepgram/community#212] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I've been getting random 499 errors back on my callback endpoint for the past few days. It usually works when I re-trigger the api call manually afterwards.
This is the request body I'm getting when this happen (instead of the transcription data):
```
{
      err_code: 'Unknown Error',
      err_msg: 'Request body not fully received',
      request_id: '1b1c9565-6283-4f55-926e-f72cfa5f8d1c'
}
```

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen?model=whisper-large&detect_language=true&punctuate=true&channels=1&callback=<callbackurl>

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

1b1c9565-6283-4f55-926e-f72cfa5f8d1c

**Comment by jtormey** [comment]
I've seen a large uptick in these errors this week as well. Any help would be appreciated, nothing on my end has changed, audio is delivered via GCS signed URL.

**Comment by bd-g** [comment]
We are aware of the issue and currently investigating. Thank you!

**Reply by bd-g** [reply, marked as answer]
A fix has been deployed.
